gradle/versions.gradle

```diff
@@ -27,7 +27,7 @@ ext.libVersions =
     jackson_annotation: '2.5.0',
     jacoco: '0.7.7.201606060606',
     snakeyaml: '1.12',
-    cassandra: '2.2.8',
+    cassandra: '3.10',
```

Comment:
This driver version is back-compat all the way to 1.2; we won't deploy a Cassandra 3.x server module for some time yet.